### PR TITLE
Add copy for self screener start

### PR DESCRIPTION
**Pull request by johnschoeman**
Why:
Currently we arent providing copy for the self screener intro screen,
which is causing CI to fail for any commit that updates the translations
file.

This commit:
Adds the copy to the self screener intro screen.

Screen Shot 2020-10-02 at 9 16 34 AM

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -257,6 +257,11 @@
     "how_the_app_works": "How the app works",
     "legal": "Legal"
   },
+  "self_screen": {
```
Comment (aledustet) [inline src/locales/en.json]: this was a typo on the `CallEmergencyServices.tsx` if we update the start of the key we can delete this one

**Review by aledustet — APPROVED** [review]
Beat me to it 😄  other than the type on the `self_screen` key this is a go for me. I'll delete it on `lokalise` so if you delete it from here we are a go.

**Comment by aledustet** [comment]
@johnschoeman I checked, no need to update lokalise, as long as we merge this one first and the other `911` updates after

**Comment by johnschoeman** [comment]
@aledustet , so i cant pass ci without including `self_screen`, is lokalize updated yet?

**Pull request merged**